include/effectmanager.h

```diff
@@ -60,13 +60,11 @@ bool ReadCurrentEffectIndex(size_t& index);
 
 std::shared_ptr<LEDStripEffect> GetSpectrumAnalyzer(CRGB color);
 std::shared_ptr<LEDStripEffect> GetSpectrumAnalyzer(CRGB color, CRGB color2);
-extern DRAM_ATTR std::unique_ptr<EffectFactories> g_ptrEffectFactories;
 
 // EffectManager
 //
 // Handles keeping track of the effects, which one is active, asking it to draw, etc.
 
-template <typename GFXTYPE>

```

Comment:
EffectManager is only ever instantiated with GFXBase as the graphics type, so it doesn't need to be a template.